include/material/newtonian.tcc

```diff
@@ -25,9 +27,9 @@ mpm::dense_map mpm::Newtonian<Tdim>::initialise_state_variables() {
 //! Compute pressure
 template <unsigned Tdim>
 double mpm::Newtonian<Tdim>::thermodynamic_pressure(
-    double volumetric_strain) const {
+    double volumetric_strain_rate) const {
   // Bulk modulus
-  return (-bulk_modulus_ * volumetric_strain);
+  return (-bulk_viscosity_ * volumetric_strain_rate);
```

Comment:
you return dp/dt? Is that what you want? I don't think so.

Reply:
nope. I am returning to stress. See the screenshot above.

Reply:
Sorry, I don't understand. Can you explain a bit more clearly what is `-bulk_density * volumetric_strain_rate`. For me it is dp/dt.

Reply:
You need to return `dp` not `dp/dt`, as we compute `dp = K * (dv / v0) ` so this should be dvolumetric_strain, not `dvolumetric_strain_rate`, Shyamini is saying exactly this, I think

Reply:
Okay. I think you are right. I will reformulate. Thank you :smiley: